nix/sources.json

```diff
@@ -0,0 +1,62 @@
+{
+    "hackage-nix": {
+        "branch": "master",
+        "description": "Automatically generated Nix expressions for Hackage",
+        "homepage": "",
+        "owner": "input-output-hk",
+        "repo": "hackage.nix",
+        "rev": "5c74d8440ce1925e6542ab028e27ac93b70a4f59",
+        "sha256": "1rfaf8zwciryshyfg02wz0q7phimi37bmp9rl1y6gl13k3pmj812",
+        "type": "tarball",
+        "url": "https://github.com/input-output-hk/hackage.nix/archive/5c74d8440ce1925e6542ab028e27ac93b70a4f59.tar.gz",
+        "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
+    },
+    "haskell-nix": {
```

Comment:
I've never actually used niv, but I'm wondering if it is a mistake to have two `haskell-nix` keys here?  I would think that only one of them would actually end up getting read.

Nix attribute sets can't have repeated keys:

```nix
$ nix repl
nix-repl> { a = 3; a = 4; }
error: attribute 'a' at (string):1:10 already defined at (string):1:3
```

Reply:
Hi @cdepillabout! Thanks for reviewing :)
Where have you spotted the second `haskell-nix` key? I can’t find it

Reply:
Oh, my bad!  I misread `hackage-nix` and `haskell-nix` as both `haskell-nix`!

Please ignore this comment :-)